### PR TITLE
Hwinfo 2

### PR DESCRIPTION
**Pull request by Pavithra1602**
fixed Pep8 compliant issue.

### CODE REVIEW

File: generic/hwinfo.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        self.run_cmd("hwinfo --tv")
+        self.run_cmd("hwinfo --uml")
+        self.run_cmd("hwinfo --usb")
+        self.run_cmd("hwinfo --usb-ctrl")
+        self.run_cmd("hwinfo --vbe")
+        self.run_cmd("hwinfo --wlan")
+        self.run_cmd("hwinfo --xen")
+        self.run_cmd("hwinfo --zip")
+        self.run_cmd("hwinfo --short")
+        self.run_cmd("hwinfo --listmd")
```
Comment (abdhaleegit) [inline generic/hwinfo.py]: a better way to handle all above options is to use list: like below

options = ['all', 'arch',  . . . . 'sys', 'usb', 'sound', 'tape', 'tv', 'scsi', 'listmd']
for ops in options:
    cmd = "hwinfo  --%s" % ops
    self.run_cmd(cmd)

File: generic/hwinfo.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        disk_name = process.system_output("df -h | egrep '(s|v)d[a-z][1-8]' | "
+                                          "tail -1 | cut -d' ' -f1",
```
Comment (abdhaleegit) [inline generic/hwinfo.py]: above command has extra double qoute, which may fail.
Why don't we use 'disk' module in avocado utils to get the disk node ? Or
Pass the disk from yaml file, which is always good.  self.disk = self.params.get('disk')

File: generic/hwinfo.py (hunk 1 above)
Comment (nasastry) [inline generic/hwinfo.py]: Already given this feedback in other PR opened.

File: generic/hwinfo.py (hunk 2 above)
Comment (nasastry) [inline generic/hwinfo.py]: In my view `double-quote` will not be an issue.

**Comment by Pavithra1602** [comment]
Raised new PR.
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/338

**Pull request closed without merging**